src/main/java/com/limechain/storage/DBRepository.java

```diff
@@ -47,6 +51,19 @@ public DBRepository(String path, String chain) {
         }
     }
 
+    private void cleanDatabaseFolder(File file) {
+        try {
+            FileUtils.cleanDirectory(file.getAbsoluteFile());
+            log.log(Level.INFO, "\uD83D\uDDD1️DB cleaned");
+
+        } catch (IOException e) {
+            log.log(Level.SEVERE, String.format("Error deleting db folder. Exception: '%s', message: '%s'",
+                            e.getCause(),
+                            e.getMessage()),
+                    e);
+        }
+    }
```

Comment:
Not sure if we should catch the error and allow the host to continue. This puts us in risk of using old DB data that can cause undefined behaviour.